### ISSUE TITLE
Unknown Network Crash Log

### ISSUE THREAD

**Issue by Gongcu** [getsentry/sentry-cocoa#2654]
### Platform

iOS

### Installed

CocoaPods

### Version

7.3.0

### Steps to Reproduce

1. network call using RxSwift zip or combineLatest ...
```swift
private func saveTokenIfTokenBeSetAndUserLoggedIn() {
    Observable.combineLatest(
      tokenStream.subscribe(on: concurrentScheduler),
      userIdStream.subscribe(on: concurrentScheduler)
    )
    .flatMap { (token, userId) in
      self.pushTokenRepository.saveToken(token, userId: userId)
    }
    .subscribe(on: concurrentScheduler)
    .subscribe(
      onNext: { _ in
        Logger.log("Token save success")
      },
      onError:{ _ in
        Logger.log("Token save failure")
      })
    .disposed(by: disposeBag)
  }
```
2. crash log
```
nw_activity_complete_with_reason_and_underlying_error_block_invoke BUG IN CLIENT OF LIBNETWORK: Cannot complete activity more than once, dumping backtrace:
0   Network                             0x00000001d0dfe748 __nw_create_backtrace_string + 192
    1   Network                             0x00000001d0f69a18 66645D9C-4336-3A4D-8060-866F18C64BD2 + 8145432
    2   Network                             0x00000001d0f5ff9c 66645D9C-4336-3A4D-8060-866F18C64BD2 + 8105884
    3   Network                             0x00000001d0f68d64 nw_activity_complete_with_reason_and_underlying_error + 220
    4   Network                             0x00000001d0f68934 nw_activity_complete_with_reason + 88
    5   CFNetwork                           0x00000001d1585828 _CFNetworkErrorCopyLocalizedDescriptionWithHostname + 21824
    6   CFNetwork                           0x00000001d155f8dc _CFNetworkErrorGetLocalizedRecoverySuggestion + 3384
    7   CFNetwork                           0x00000001d15a348c CFHTTPMessageCopyRequestURL + 6320
    8   CFNetwork                           0x00000001d155fbe0 _CFNetworkErrorGetLocalizedRecoverySuggestion + 4156
    9   libdispatch.dylib                   0x000000010199c53c _dispatch_call_block_and_release + 32
    10  libdispatch.dylib                   0x000000010199dff0 _dispatch_client_callout + 20
    11  libdispatch.dylib                   0x00000001019a60a0 _dispatch_lane_serial_drain + 988
    12  libdispatch.dylib                   0x00000001019a6dfc _dispatch_lane_invoke + 472
    13  libdispatch.dylib                   0x00000001019b3cac _dispatch_workloop_worker_thread + 740
    14  libsystem_pthread.dylib             0x000000021e640df8 _pthread_wqthread + 288
    15  libsystem_pthread.dylib             0x000000021e640b98 start_wqthread + 8
```

reproduce with latest sentry version 
and I'm using Xcode 14 and Xcode 14.2 with rosseta.
The Log did not appear when opt out Sentry
### Expected Result

no crash log

### Actual Result

crash log

**Comment by philipphofmann** [comment]
Please update to the latest Cocoa 7.x version and reopen if this issue persists. This should already be fixed with [7.5.3](https://github.com/getsentry/sentry-cocoa/blob/main/CHANGELOG.md#753).

**Comment by Gongcu** [comment]
I mistyped the version. I'm using 7.30.1, not 7.3.0. The bug is reproduced in this version.
